eskipfile/remote_test.go

```diff
@@ -91,6 +91,13 @@ func TestLoadAll(t *testing.T) {
 		t.Run(test.title, func(t *testing.T) {
 			options := &RemoteWatchOptions{RemoteFile: s.URL, Threshold: 10, Verbose: true, FailOnStartup: true}
 			client, err := RemoteWatch(options)
+			defer func() {
+				c, ok := client.(*remoteEskipFile)
+				if ok {
+					c.Close()
+					time.Sleep(time.Second)
```

Comment:
I think this sleep is redundant

Reply:
I thought it fixes the goroutines cleanup, because Close() is normally async: close(quitCH) and then it has to schedule the goroutine in the `for{ select {} }` loop...